### PR TITLE
Catalog Docs for 0.2

### PR DESCRIPTION
**Pull request by jbouffard**
This PR adds a notebook that goes over the `catalog` module.

This PR is based on another, open PR #457

### CODE REVIEW

**Comment by jbouffard** [comment]
@echeipesh I wasn't able to get the `proj_query` to work, but otherwise this section of the catalog docs is complete.

File: docs/notebook-demos/catalog.ipynb [hunk 1]
```diff
@@ -0,0 +1,614 @@
...
+    "          time_unit=gps.TimeUnit.SECONDS)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Saving a Pyramid\n",
+    "\n",
+    "For those that are unfamiliar with the `Pyramid` class, please see the [Pyramid section] of the visualization guide. Otherwise, please continue on.\n",
```
Comment (rossbernet) [inline docs/notebook-demos/catalog.ipynb]: will this link work? I know you said you were changing how the docs are structured now

File: docs/notebook-demos/catalog.ipynb [hunk 2]
```diff
@@ -0,0 +1,614 @@
...
+    "          layer_zoom=11)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Querying a Layer\n",
+    "\n",
+    "When only a certain section of the layer is interest, one can retrieve these areas of the layer through the `query` method. Depending on the type of data being queried, there are a couple of ways to filter what will be returned."
```
Comment (rossbernet) [inline docs/notebook-demos/catalog.ipynb]: *is of interest

File: docs/notebook-demos/catalog.ipynb [hunk 3]
```diff
@@ -0,0 +1,614 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# In addition to Polygons, one can also querry using MultiPolygons.\n",
```
Comment (rossbernet) [inline docs/notebook-demos/catalog.ipynb]: `querry` - 1 `r`

File: docs/notebook-demos/catalog.ipynb (hunk 1 above)
Comment (jbouffard) [inline docs/notebook-demos/catalog.ipynb]: That link will at some point work. the `Pyramid` class will be discuessed in more detail in the visualization guide, which has yet to be made.

File: docs/notebook-demos/catalog.ipynb (hunk 3 above)
Comment (jbouffard) [inline docs/notebook-demos/catalog.ipynb]: Sorry, I'm not sure what you mean by this.

Comment (jpolchlo) [inline docs/notebook-demos/catalog.ipynb]: He means "querry" => "query"

Comment (jbouffard) [inline docs/notebook-demos/catalog.ipynb]: Ah, okay. I'll fix that now.

**Pull request merged**